File: demo/src/java/ast/KoKoString.java
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.ast;
import java.util.*;
import java.io.*;

public class KoKoString extends KoKoAtom<String>{
   
   public KoKoString(String value){
      super(value);
   }

   @Override
   public KoKoValue eval(KoKoContext ctx){
	   return new KoKoStringValue(getValue());  
   }
}

File: KoKoslan/src/kotlin/ast/KoKoString.kt
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.kt.ast
import java.util.*
import java.io.*

class KoKoString(value : String) : KoKoAtom<String>(value) {
    
   override fun eval(ctx : KoKoContext) : KoKoValue {
	   return KoKoStringValue(getValue())  
   }
}